src/main/resources/org/broadinstitute/hellbender/utils/plotter/ACNVResultsPlottingPerSeg.R

```diff
@@ -0,0 +1,88 @@
+#NOTE: the Java wrapper for this script first sources CNV_plotting_library.R
```

Comment:
There's a lot of code duplication.  Genome plotting and per-segment plotting should be a single script with a boolean parameter to toggle per-segment mode.